command/ca/eak/eak.go

```diff
@@ -0,0 +1,49 @@
+package eak
+
+import (
+	"encoding/base64"
+
+	adminAPI "github.com/smallstep/certificates/authority/admin/api"
+	"github.com/smallstep/certificates/ca"
+	"github.com/urfave/cli"
+)
+
+type cliEAK struct {
+	id   string
+	name string
+	key  string
+}
+
+func toCLI(ctx *cli.Context, client *ca.AdminClient, eak *adminAPI.CreateExternalAccountKeyResponse) (*cliEAK, error) {
+	// TODO: more fields for other purposes, like including the createdat/boundat/account for listing?
+	return &cliEAK{id: eak.KeyID, name: eak.Name, key: base64.StdEncoding.EncodeToString(eak.Key)}, nil
+}
+
+// Command returns the jwk subcommand.
+func Command() cli.Command {
+	return cli.Command{
+		Name:      "eak",
```

Comment:
eak is not a common name. Yes, it means external account key. EAB is a more common name (google `acme eak` vs `acme eab`), and it's probably best understood.

So I think we should probably add commands under `step ca eab` or `step ca acme-eab` or even `step ca acme eab`

The same can be applied to APIs in step-ca.

Reply:
I've added an `acme` command and `eab` subcommand below that.